main.go

```diff
@@ -83,10 +86,20 @@ func connectDiscordBot() (*discordgo.Session, error) {
 
 func presenceUpdate(s *discordgo.Session, m *discordgo.PresenceUpdate) {
 	info, err := Server.Info()
+	
 	if err != nil {
 		fmt.Println(err)
 		return
 	}
+	
+	if first == true {
+		first = false
+		LastInfo = *info
```

Comment:
This does some heavy code duplication.

Why not instead alter `if LastInfo.Players == info.Players {` to be `if LastInfo.Players == info.Players || first {` to avoid the duplication? Just always set `first` to false at the end of the function.

Reply:
Because the suggested statement will evaluate as true as soon as ```LastInfo.Players == info.Players``` evaluates as true, which is exactly what happens at startup anyway (if it didn't, there would be no need for the ```initializedPresence``` variable. Proper solution should be ```if LastInfo.Players == info.Players && initializedPresence``` without ever resetting the variable to false.